src/main/scala/io/archivesunleashed/app/CommandLineApp.scala

```diff
@@ -117,6 +117,9 @@ class CommandLineApp(conf: CmdAppConf) {
         if (!configuration.outputFormat.isEmpty && configuration.outputFormat() == "GEXF") {
           new File(saveTarget).mkdirs()
           WriteGEXF(DomainGraphExtractor(rdd), Paths.get(saveTarget).toString + "/GEXF.xml")
+        } else if (!configuration.outputFormat.isEmpty && configuration.outputFormat() == "GRAPHML") {
+          new File(saveTarget).mkdirs()
+          WriteGraphML(DomainGraphExtractor(rdd), Paths.get(saveTarget).toString + "/GRAPHML.xml")
```

Comment:
First, 🤦 

I realize that I've been so focused on looking at the actual innards of the file using a text editor, that I was blind to the fact that I would have to rename the extensions in order to load them into Gephi.

So probably`WriteGraphML` here should actually be writing to a `.graphml` file. And then same with the `WriteGEXF` files. With the correct extension, they load natively into Gephi; if not, you have to manually rename them.  

I changed the `WriteGraphML` and `WriteGEXF` lines to the following, and it worked (same output, just loads without having to rename):

```scala
WriteGEXF(DomainGraphExtractor(rdd), Paths.get(saveTarget).toString + "/GEXF.gexf")
```

and

```scala
WriteGraphML(DomainGraphExtractor(rdd), Paths.get(saveTarget).toString + "/GRAPHML.graphml")
```

Again, I can't believe I didn't catch this earlier, having spent like 30 minutes in the files themselves.. just looking at the details rather than the giant file name staring me in the face.